datasets/pragmeval/pragmeval.py

```diff
@@ -0,0 +1,430 @@
+      booktitle ="Proceedings of the 17th Annual Meeting of the Special Interest Group on      Discourse and Dialogue    ",
+      year ="2016",
+      publisher ="Association for Computational Linguistics",
+      pages ="31--41",
+      location ="Los Angeles",
+      doi ="10.18653/v1/W16-3604",
+      url ="http://aclweb.org/anthology/W16-3604"
+    }
+    """,
+    "squinky": """
+    @article{DBLP:journals/corr/Lahiri15,
+      author    = {Shibamouli Lahiri},
+      title     = {{SQUINKY! A Corpus of Sentence-level Formality, Informativeness,
+                   and Implicature}},
+      journal   = {CoRR},
+      volume    = {abs/1506.02306},
+      year      = {2015},
+      url       = {http://arxiv.org/abs/1506.02306},
+      timestamp = {Wed, 01 Jul 2015 15:10:24 +0200},
+      biburl    = {http://dblp.uni-trier.de/rec/bib/journals/corr/Lahiri15},
+      bibsource = {dblp computer science bibliography, http://dblp.org}
+    }
+    """,
+    "verifiability": """@inproceedings{park2014identifying,
+      title={Identifying appropriate support for propositions in online user comments},
+      author={Park, Joonsuk and Cardie, Claire},
+      booktitle={Proceedings of the first workshop on argumentation mining},
+      pages={29--38},
+      year={2014}
+    }""",
+    "emobank": """"
+    @inproceedings{buechel-hahn-2017-emobank,
+        title = "{E}mo{B}ank: Studying the Impact of Annotation Perspective and Representation Format on Dimensional Emotion Analysis",
+        author = "Buechel, Sven  and
+          Hahn, Udo",
+        booktitle = "Proceedings of the 15th Conference of the {E}uropean Chapter of the Association for Computational Linguistics: Volume 2, Short Papers",
+        month = apr,
+        year = "2017",
+        address = "Valencia, Spain",
+        publisher = "Association for Computational Linguistics",
+        url = "https://www.aclweb.org/anthology/E17-2092",
+        pages = "578--585",
+        abstract = "We describe EmoBank, a corpus of 10k English sentences balancing multiple genres, which we annotated with dimensional emotion metadata in the Valence-Arousal-Dominance (VAD) representation format. EmoBank excels with a bi-perspectival and bi-representational design. On the one hand, we distinguish between writer{'}s and reader{'}s emotions, on the other hand, a subset of the corpus complements dimensional VAD annotations with categorical ones based on Basic Emotions. We find evidence for the supremacy of the reader{'}s perspective in terms of IAA and rating intensity, and achieve close-to-human performance when mapping between dimensional and categorical formats.",
+    }
+    """,
+}
+
+
+def get_labels(task):
+    folder_name = TASK_TO_FOLDER[task]
+    label_classes = requests.get(
+        f"https://raw.githubusercontent.com/disceval/DiscEval/master/disceval/{folder_name}/labels"
+    ).text.split("\n")
+    label_classes = [x.strip() for x in label_classes if x.strip()]
+    return label_classes
```

Comment:
Thank you ! I think it is indeed a nice 

> Really cool ! Thanks for adding this one :)
> Good job at adding all those citations for each task
> 
> Looks like the dummy data test doesn't pass. Maybe some files are missing in the dummy_data.zip files ?
> The error reports `pragmeval/verifiability/train.tsv` to be missing
> 
> Also could you add the tags part of the dataset card (the rest is optional) ?
> See more info here : https://github.com/huggingface/datasets/blob/master/ADD_NEW_DATASET.md#tag-the-dataset-and-write-the-dataset-card

Thank you, I think it will be a nice addition to the library since it has ~20 subtasks. I did the dummy and dataset_infos manually and I finally figured out that the cli helper can automate it (see last commit)

Reply:
> Can you include the labels directly here in the script ?
> We should avoid the use of external requests besides the data files by the `dl_manager`

Some tasks have many labels (more than 20), so it could harm readability a bit. There are many tasks as well (~ counting the subtasks)
Shall I still add them in hard text ? Or use dedicated files ?

Reply:
It's fine to have them in the script directly.
For readability, feel free to declare the variables of the labels lists before the dataset class definition. This way the _info() function will still be easily readable